src/App.js

```diff
@@ -1,13 +1,40 @@
 import React from 'react';
+import TodoList from './components/TodoComponents/TodoList';
 
 class App extends React.Component {
   // you will need a place to store your state in this component.
   // design `App` to be the parent component of your application.
   // this component is going to take care of state, and any change handlers you need to work with your state
+  todoItems = 
+  [
+    {
+      task: 'Organize Garage',
+      id: 1528817077286,
+      completed: false
+    },
+    {
+      task: 'Bake Cookies',
+      id: 1528817084358,
+      completed: false
+    }
+  ]
+
```

Comment:
Adding data this way works, but there are more efficient ways of doing it, here you only have two objects but what if you had 50 objects would you do it the same way.